app/models/ems_refresh/parsers/kubernetes.rb

```diff
@@ -341,8 +341,17 @@ def parse_container(container, pod_id)
         :backing_ref     => container.containerID,
         :container_image => parse_container_image(container.image, container.imageID)
       }
-
-      # TODO, state
+      Hash(container.state).each do |key, _val|
```

Comment:
convert to hash:
container.state !=  Hash(container.state) since container.state is of type Kubeclient::pod and container.state.each Is empty. is there a better way to extract data?.

As for the iteration, the returned data looks like so:
"state": {
  "waiting": {
    "reason": "Image: openshift/origin-deployer:v1.0.4 is ready, container is creating"
  }
I could just as well use first instead of each, but I need to get the key inside state due to the nesting.